package org.sus.domain.cidadao.exception;

public class CidadaoNaoEncontradoException extends RuntimeException {
    public CidadaoNaoEncontradoException(String message) {
        super(message);
    }
}
